addons/sale/models/sale_order.py

```diff
@@ -253,12 +253,18 @@ def _rec_names_search(self):
         comodel_name='payment.transaction',
         relation='sale_order_transaction_rel', column1='sale_order_id', column2='transaction_id',
         string="Transactions",
+        groups='account.group_account_invoice',
```

Comment:
This doesn't make any sense to me... Why would you need the invoicing rights to see the payment transactions of your SO?

Reply:
But you don't need to read in order to know that there are records, right?

And why is it not implicit? What if someone changes the access rules? (including changes done in db)